usermods/audioreactive/audio_reactive.h

```diff
@@ -300,7 +300,7 @@ void FFTcode(void * parameter)
 #endif
 
 #ifdef UM_AUDIOREACTIVE_USE_NEW_FFT
-      FFT.majorPeak(FFT_MajorPeak, FFT_Magnitude);                // let the effects know which freq was most dominant
+      FFT.majorPeak(&FFT_MajorPeak, &FFT_Magnitude);                // let the effects know which freq was most dominant
```

Comment:
we should contribute this upstream?  or I wonder if I merge the branch to a later version if they've found and fixed this

Reply:
Looks like they've done something similar already: https://github.com/SteveEisner/WLEDtubes/commit/ecc9443677e1c69291138e8bf8f54d524614a144

Reply:
I've just merged this into the WLEDtubes/main branch, so you should be able to rebase/merge your PR branch and remove this fix